server/app/views/components/ProgramCardFactory.java

```diff
@@ -185,25 +183,20 @@ private DivTag renderProgramRow(
                                 .with(programRow.extraRowActions()))));
   }
 
-  private ProgramDefinition getDisplayProgram(ProgramCardData cardData) {
+  private static ProgramDefinition getDisplayProgram(ProgramCardData cardData) {
     if (cardData.draftProgram().isPresent()) {
       return cardData.draftProgram().get().program();
     }
     return cardData.activeProgram().get().program();
   }
 
-  private static Instant extractLastUpdated(ProgramCardData cardData) {
-    // Prefer when the draft was last updated, since active versions should be immutable after
-    // being published.
-    if (cardData.draftProgram().isEmpty() && cardData.activeProgram().isEmpty()) {
-      throw new IllegalArgumentException("Program neither active nor draft.");
-    }
-
-    ProgramDefinition program =
-        cardData.draftProgram().isPresent()
-            ? cardData.draftProgram().get().program()
-            : cardData.activeProgram().get().program();
-    return program.lastModifiedTime().orElse(Instant.EPOCH);
+  public static Comparator<ProgramCardData> comparator() {
```

Comment:
Can we clarify what fields comparator compares by? lastModifiedTimeThenNameComparator()? We are in java world after all and can use all the words.

Reply:
Done.